### ISSUE TITLE
Can't type the '+' character on PET 8296 (and other keyboard mapping woes)

### ISSUE THREAD

**Issue by LeifBloomquist** [ChartreuseK/PETTERM#6]
I'm trying to use petterm 5.0 on my PET 8296 and [Stupid PET Tricks wifi modem](https://www.bit-zeal.com/product/stupid-pet-tricks-module/3) and it's soo close!

I'm assuming I should be using **petterm80B.prg** - (I tried petterm80G.prg just in case, and the keyboard didn't work at all)

So the command to get into the Wifi settings is 'at+config' but I can't type the + character.  I get a '|' instead.

Other mismatched mappings:

```
[ gives ]
\ gives @ 
@ gives [
] gives ;
; gives \
```

All the other keys seem to work as expected.  Would there need to be a special version for the 8296 keyboard, or as there something else I could try?

**Comment by LeifBloomquist** [comment]
I've created a fork with a new keyboard mapping for the PET 8296 that fixes this issue.  I'll submit a pull request once I've tested it a little more.

https://github.com/ChartreuseK/PETTERM/compare/master...LeifBloomquist:PETTERM:8296-keyboard

**Comment by LeifBloomquist** [comment]
Pull Request submitted.

https://github.com/ChartreuseK/PETTERM/pull/10

**Comment by ChartreuseK** [comment]
Sorry for not replying sooner. I didn't realize the 8296 actually uses a different keyboard mapping. I had assumed that the business and graphics keyboards were the only ones in use. Really starting to get crazy with the number of different compiled versions now for each machine.

**Comment by LeifBloomquist** [comment]
Right.   In theory you could have one executable that contains all the mappings and detects the PET model at startup (somehow)? but this approach keeps the binaries small.

**Comment by bozimmerman** [comment]
Actually, I've been experimenting with PETTERM 0.5 on both an 8032 and 2001-32B the last few days, and in BOTH cases, the + sign is not available on its normal key.  Instead, I have to press shift-] .   In other words, this might be all US business keyboards?
